.github/workflows/changelog.yml

```diff
@@ -36,9 +36,9 @@ jobs:
           commit-message: update changelog
           committer: OSC ROBOT <osc.robot@gmail.com>
           author: osc-bot <osc.robot@gmail.com>
-          branch: osc-bot/changelog-update
```

Comment:
I think I introduced a bug here where both these workflows update the same branch which may cause issues.